src/luarocks/cmd.lua

```diff
@@ -457,23 +457,21 @@ function cmd.run_command(description, commands, external_namespace, ...)
    local cmd_modules = {}
    for name, module in pairs(commands) do
       local pok, mod = pcall(require, module)
-      if pok and type(mod) == "table" then
-         if not mod.add_to_parser then
-            mod.add_to_parser = function(parser)
-               parser:command(name, mod.help, util.see_also())
-                     :summary(mod.help_summary)
-                     :handle_options(false)
-                     :argument("input")
-                     :args("*")
-            end
-            local original_command = mod.command
-            if original_command then
+      if pok and type(mod) == "table" thenend
```

Comment:
`thenend` is invalid

Reply:
doh. I meant to send this as a "suggestion" in github. Looks like it did a full PR.